src/Location/Location.tsx

```diff
@@ -12,107 +12,115 @@ import wind from '../assets/wind.png'
-// 									className="small-weather-icon"/> 
-// 									<p className='type'>Visibility</p>
-// 									<p className='unit'>{currentWeather.visibility / 5280}mi</p>
-// 							</td>
-// 						</tr>
-// 					</tbody>
-// 				</table>
-// 			</div>
-// 			<div className='info-box hourly-forecast'>
-// 				<h4 className='info-box-header'>HOURLY FORECAST</h4>
-// 			</div>
-// 			<div className='info-box weekly-forecast'>
-// 				<h4 className='info-box-header'>WEEKLY FORECAST</h4>
-// 			</div>
 
-// 		</section>
-// 	)
-// }
+	// const renderWeeklyForecast = 
+	// 	weeklyWeather.daily.map((day => {
+	// 		return <article key={day.weather.id}className="daily-weather">
+	// 			<h3 className="daily-weather-highlight">
+	// 				{day.temp.min}F/{day.temp.max}F
+	// 			</h3>
+	// 			<p className="daily-weather-icon">
+	// 				<span>{day.weather.icon}</span>
+	// 			</p>
+	// 			{/* <p className="daily-weather-minmax">{time method to check time and return temp off time need to do the same bellow for feels like}</p> */}
+	// 			<p className="daily-weather-feelslike">
+	// 			add time checker to display feels like
+	// 			</p>
+	// 			<p className="daily-pressure">{day.pressure}</p>
+	// 			<p className="daily-humidity">{day.humidity}</p>
+	// 			<p className="daily-wind">
+	// 				{/* need functionality to evaluate wind direction */}
+	// 				{day.wind_speed}   {day.wind_deg}
+	// 			</p>
+	// 			<p>{day.uvi} {day.clouds}</p>
+	// 			{/* <p>conditional that checks value of weather.main to give back data if available or return null<p> */}
+	// 		</article> 
+	// 	}))
 
-// export default Location
\ No newline at end of file
+
+	return (
+		<section className="location-section">
+			<h2 className='current-city'>{locationData.locality}</h2>
+			<h3 className='current-region'>{locationData.region}, {locationData.country_code}</h3>
+			<div className='info-box'>
+				<h4 className='info-box-header'>HAPPENING NOW</h4>
+				<p className='current-date'>{Date.now().toLocaleString()}</p>
+				<article className='current-weather'>
+					<div className='current-weather-left'>
+						<h5 className='current-temp'>{currentWeather.temp}&deg;</h5>
+						<img src={cloudyNight} alt='clouds covering moon for current sky' className='large-weather-icon'/>
+					</div>
+					<div className='current-weather-right'>
+						<span className='current-weather-right-top'>
+							<img src={wind} alt='wind icon for wind speed direction' className='small-weather-icon'/> 
+							<p className='current-wind'>{currentWeather.wind_speed} mph / {currentWeather.wind_deg}</p>
+						</span>
+						<span className='current-weather-right-bottom'>
+							<img src={waterdrop} alt='rain droplet icon for precipitation' className='small-weather-icon'/> 
+							<p className='current-precipitation'>{'Coming Soon'}</p>
+						</span>
+					</div>
+
+				</article>
+			</div>
+			<div className='info-box air-quality'>
+				<h4 className='info-box-header'>AIR QUALITY</h4>
+				<table>
```

Comment:
What are your thoughts of me slimming down the JSX here, didn't want to go all cowboy and modify what has already been built?

Reply:
Sure thing!